modules/openxBidAdapter.js

```diff
@@ -1,18 +1,30 @@
 const bidfactory = require('src/bidfactory.js');
 const bidmanager = require('src/bidmanager.js');
 const adloader = require('src/adloader');
+const ajax = require('src/ajax');
 const CONSTANTS = require('src/constants.json');
 const utils = require('src/utils.js');
 const adaptermanager = require('src/adaptermanager');
 
 const OpenxAdapter = function OpenxAdapter() {
   const BIDDER_CODE = 'openx';
   const BIDDER_CONFIG = 'hb_pb';
+  const BIDDER_VERSION = '1.0.1';
   let startTime;
+  let timeout = $$PREBID_GLOBAL$$.bidderTimeout;
```

Comment:
I didn't notice that you're overwriting the timeout below from the bidderRequest params.  That should be accurate. :+1: